zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -105,6 +105,16 @@ def query_children_volume(logical_pool, volume, snapshot, is_snapshot=False):
 def is_support_get_volume_clients():
     return shell.run("%s list client --help | grep -E '\--path'" % ZBS_BIN_PATH) == 0
 
+def is_volume_exist(logical_pool, volume):
+    o = query_volume_info(logical_pool, volume)
+    ret = jsonobject.loads(o)
+    if ret.error.code != 0:
+        match = re.search(r"status code:\s*(\w+)", ret.error.message)
```

Ruff (0.14.10)

`re` may be undefined, or defined from star imports

(F405)